### PR TITLE
fixed comments, sorted arch, added alch/bs/tailor recipes

### PR DESCRIPTION
**Pull request by gitenarra**
was fixed leatherworking and tailoring item names
sorted archaeology recipes (added categories via races)
added alchemy, blacksmithing, tailoring recipes links

### CODE REVIEW

**Comment by gitenarra** [comment]
fix punctuation mistakes

File: AtlasLootClassic_Crafting/data-cata.lua [hunk 1]
```diff
@@ -1492,34 +1496,34 @@ data["LeatherworkingCata"] = {
...
+				{ 5, 85067 },	-- Dragonbone Leg Reinforcements
+				{ 6, 78477 },	-- Dragonscale Leg Armor
+				{ 7, 78419 },	-- Scorched Leg Armor
+				{ 9, 101600 },	-- Drakehide Leg Reinforcements
+				{ 10, 101599 },	-- Drakehide Leg Armor
 				{ 16, 85007 },	-- Draconic Embossment - Stamina
 				{ 17, 85008 },	-- Draconic Embossment - Agility
 				{ 18, 85009 },	-- Draconic Embossment - Strength
 				{ 19, 85010 },	-- Draconic Embossment - Intellect
-				{ 21, 85067 },	-- Dragonbone Leg Reinforcements
```
Comment (MikiDylewski) [inline AtlasLootClassic_Crafting/data-cata.lua:1520]: Any particular reason for these to be deleted?

Comment (snowflame0) [reply AtlasLootClassic_Crafting/data-cata.lua:1520]: Just rearranged it looks like.

**Review by snowflame0 — APPROVED** [review]
Looks good, thank you again.

**Pull request merged**